Bottom: 85e07ae84b10661f6c4c5279d4210e52d1ac5761
Top:    596cce94f0a29fb9caa10592d6e2623f41ed3027
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Fix 3D contour export


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a64a451..dd071eb 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -266,7 +266,7 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
       *cx = COORD_TO_STEP_X (PCB, contour->cx);
       *cy = COORD_TO_STEP_Y (PCB, contour->cy);
       *r = COORD_TO_MM (contour->radius);
-      *cw = (contour->Flags.orient != PLF_DIR);
+      *cw = (contour->Flags.orient == PLF_INV);
     }
 #endif
 }
